tests/test_http.py

```diff
@@ -1077,6 +1077,7 @@ class TestBatch(unittest.TestCase):
 
   def setUp(self):
     model = JsonModel()
+    self.MAX_BATCH_LIMIT = 1000
```

Comment:
No point in adding an attribute in `setUp` to be used only in one testcase:  just import the constant in that method (see suggestion below).

Reply:
Thanks for the suggestion! Committed the change.